dts/vendor/nordic/nrf54h20.dtsi

```diff
@@ -151,6 +151,7 @@
 	reserved-memory {
 		#address-cells = <1>;
 		#size-cells = <1>;
+		ranges = <>;
```

Comment:
Why not just `ranges;` like in other nRF SoCs (and devicetree specification)?

Reply:
that would look like a boolean property, this is unmistakable :)